config/sets/symfony/symfony4/symfony44.php

```diff
@@ -2,11 +2,121 @@
 
 declare(strict_types=1);
 
+use Rector\Arguments\Rector\ClassMethod\ReplaceArgumentDefaultValueRector;
+use Rector\Arguments\ValueObject\ReplaceArgumentDefaultValue;
 use Rector\Config\RectorConfig;
 
 // https://github.com/symfony/symfony/blob/4.4/UPGRADE-4.4.md
 
 return static function (RectorConfig $rectorConfig): void {
+    $rectorConfig->ruleWithConfiguration(ReplaceArgumentDefaultValueRector::class, [
```

Comment:
This should import file

`symfony44-weblink.php` then include here and in Set class

https://github.com/rectorphp/rector-symfony/blob/main/src/Set/SetProvider/Symfony4SetProvider.php